packages/core-utils/src/query.js

```diff
@@ -34,6 +34,8 @@ export const defaultParams = [
  */
 const TIME_FORMATS = [
   "HH:mm:ss",
+  "HH:mm",
```

Comment:
Move `OTP_DATE_TIME_FORMAT` here or delete that last entry.

Reply:
Good catch thanks! Have opted to remove as the constant is used as a fallback anyway.